.github/workflows/android-beta.yaml

```diff
@@ -81,10 +81,25 @@ jobs:
         run: |
           cmake --version
           ninja --version
+          echo "ANDROID_CODE=$(../tools/unix/version.sh android_code)" >> "$GITHUB_ENV"
           ./gradlew -Pfirebase assembleGoogleBeta uploadCrashlyticsSymbolFileGoogleBeta uploadCrashlyticsMappingFileGoogleBeta
 
       - name: Upload beta apk to App Distribution
         shell: bash
         working-directory: android
         run: |
           ./gradlew appDistributionUploadGoogleBeta
+
+      - name: Upload beta apk to GitHub artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: OrganicMaps-${{ env.ANDROID_CODE }}-google-beta.apk
+          path: android/app/build/outputs/apk/google/beta/OrganicMaps-${{ env.ANDROID_CODE }}-google-beta.apk
+          if-no-files-found: error
+
+      - name: Upload beta debug symbols to GitHub artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: native-debug-symbols.zip
+          path: android/app/build/outputs/native-debug-symbols/googleBeta/native-debug-symbols.zip
+          if-no-files-found: error
```

Comment:
Why do you want to upload symbols? Who (and how) is going to use them?

Reply:
Me, when reading crash logs from users without Firebase AppTester installed.

Reply:
I'm pretty sure it doesn't matter how users got the app, all crashes would still be reported to Firebase

Reply:
Even if the phone is degoogled or in a Google-blocked country? Are you 100% sure? The APK is needed for easier distribution anyway, so there's no harm in storing debug symbols with it. What are your concerns?

Reply:
fair point